pollination_apps/cli/__init__.py

```diff
@@ -175,7 +175,10 @@ def new():
 @click.argument('owner', type=click.STRING)
 @click.option('-n', '--name', help='the name of the app (defaults to folder name)')
 @click.option('-t', '--tag', help='the tag for this version of the app')
-def run(path, owner, name, tag):
+@click.option('--static/--editable', ' /-e', help='Flag to note whether the container should '
+              'be editable by mounting the app path as a volume to the docker container. ', 
+              default=False, show_default=True)
```

Comment:
What if we use this as a single argument flag? I'm not sure if the default should be set to False or True but you can test it on your end.

```suggestion
@click.option('-e', '--editable', help='An option to set the container to be editable '
              'by mounting the app path as a volume to the docker container. ', 
              default=False, show_default=True, is_flag=True)
```